src/main/java/SimpleFileSystem/Main.java

```diff
@@ -41,28 +42,24 @@ public Main() {}
     public static void main(String[] args) {
```

Comment:
I thing this class can be removed complete it doesn't seem to be used anywhere.

Reply:
By thing I mean the whole class

Reply:
addresed here 067d2b19